### ISSUE TITLE
Index-Discovery commands not listed

### ISSUE THREAD

**Issue by paulo-graca** [DSpace/DSpace#3258]
**Describe the bug**
It seams `index-discovery` is missing or omitted from the CLI list.

**To Reproduce**
1. When executing `[/dspace]/bin/dspace ` a list is displayed

**Expected behavior**
In the displayed list, it's missing `index-discovery`, but regardless, if you try to execute it the process script will be executed.
```
# /dspace/bin/dspace index-discovery -b
The script has started
(Re)building index from scratch.

```

**Comment by tdonohue** [comment]
@paulo-graca : Are you looking into this (as I see you assigned it to yourself)?  It looks like it's related to moving these scripts into processes (which can be triggered from the UI or commandline) and this work #2596 

If you are not working on this, let me know & I'll reassign it.

**Comment by paulo-graca** [comment]
Sorry @tdonohue , no, I'm swamped.

**Comment by tdonohue** [comment]
@benbosman : assigning this to you for your team to analyze & estimate. It looks to be related to #2596

**Comment by tdonohue** [comment]
Closing, as this was fixed by #3278 (but accidentally not linked up properly before merging)
